python/mxnet/visualization.py

```diff
@@ -252,6 +252,10 @@ def plot_network(symbol, title="plot", save_format='pdf', shape=None, node_attrs
         shape_dict = dict(zip(interals.list_outputs(), out_shapes))
     conf = json.loads(symbol.tojson())
     nodes = conf["nodes"]
+    # check if multiple nodes have the same name
+    if len(nodes) != len(set([node["name"] for node in nodes])):
+        logging.warning("There are multiple variables with the same name in your graph, "
```

Comment:
It would good to print the duplicate name in your warning message.

Reply:
Second this.